packages/reakit/src/Menu/Menu.tsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
```

Comment:
```suggestion
import * as React from "react";
```

Reply:
Happy to amend here and below 🙌🏻

Out of curiosity, why there's a need to access all the exports of **React** behind a namespace? is it a convention or something related to **Typescript**? 

Can it be something we could set the linter to catch? ( happy to add that config in this PR if _positivie_ )

Reply:
That is due to not having `allowSyntheticDefaultImports` on our `tsconfig.json` file. I don't know the implications of having or not this, but i know for sure that the lack of this implies importing things with `*`.

Reply:
It would throw errors on users' projects (which is weird, but ok). See https://github.com/reakit/reakit/issues/198#issuecomment-414009701